### DISCUSSION TITLE
Security Onion Logs Retention Period

### DISCUSSION THREAD

**Discussion by esbat-max** [Security-Onion-Solutions/securityonion#5835] [Unsupported Versions] [answered]
Dears,

I am unable to view the logs in security onion -> Kibana -> Discover page for more than 2 days. I checked the storage of the system is has enough  to store logs for more than a month.

I am using SO in standalone mode, version 2.3.7.

any advise ?

**Comment by dougburks** [comment, marked as answer]
What options did you choose in Setup?

How much disk space do you have?

**Reply by esbat-max** [reply]
Hello Dough,

Thank you for your questions,

Please find below the answers:

1. What options did you choose in Setup?
     
   Please clarify
- Security onion is operating in standalone mode, 
- one network interface

2. How much disk space do you have?
[ ~]$ df -H
Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                  34G     0   34G   0% /dev
tmpfs                     34G  349k   34G   1% /dev/shm
tmpfs                     34G  213M   34G   1% /run
tmpfs                     34G     0   34G   0% /sys/fs/cgroup
/dev/mapper/system-root   70G   21G   49G  30% /
/dev/mapper/system-tmp   2.1G   34M  2.1G   2% /tmp
_/dev/mapper/system-nsm   296G   80G  216G  27% /nsm_
/dev/sda2                521M  197M  324M  38% /boot

Index size for the last 48 hours:
![image](https://user-images.githubusercontent.com/66309463/137075012-8aa6230b-539a-45d4-b382-56910383e3ce.png)

Log sources connected until now Domain controllers and Exchange servers for the last 48hrs:
Exchange 1              28,889,788
Exchange 2                22,082,412
UG                             3,364,477
DC1                           2,850,612
DC2                           1,205,535
DC3                           1,103,601
DC4                            394,786

I did some fine tuning to the curator config file and increase the **log_size_limit to 200GB** and  **cur_close_days:100**, So, the curator wont take any action and deleted them.

In my humble opinion, the problem is solved.

Please advise of any mistake from our side.

Br,
E

**Reply by dougburks** [reply]
> * Security onion is operating in standalone mode,
> * one network interface

Standalone usually requires two network interfaces: one for management and one for sniffing. If you're not doing any network sniffing, then you probably don't need or want standalone mode.
https://docs.securityonion.net/en/2.3/hardware.html#nic
https://docs.securityonion.net/en/2.3/architecture.html

> **cur_close_days:100**, So, the curator wont take any action and deleted them.

To be clear, `cur_close_days` only controls when Curator closes indices as deletion is handled separately. Also, please be aware that having 100 days of open indices may cause performance issues if the system isn't properly tuned.
https://docs.securityonion.net/en/2.3/curator.html#configuration
